SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security policy
+
+Thanks for helping make nuntius safer for everyone.
+
+Find below updated information on our security policy.
+
+## Security
+
+We take the security of this software seriously.
+
+We don't implement a bug bounty program or bounty rewards, but will work with
+you closed to ensure that your findings gets the appropriate handling.
```

Comment:
"closed"?

Reply:
Yeah, it should be "closely". Copy-paste means this error is found elsewhere too 🙄

Reply:
Removed "closed", changed "findings gets" to "findings get".
Rebased onto the main branch, to resolve merge conflicts, and force-pushed.

Reply:
a167d0d